environment.yml

```
@@ -6,7 +6,7 @@ channels:
   - aps-anl-tag
 dependencies:
   - python>=3.6
-  - bluesky
+  - bluesky>=1.6.7
```

Conda folks like to add a blank space between the package name and the version pin.
```suggestion
  - bluesky >=1.6.7
```

The `python>=3.6` line above may need a similar update.